### DISCUSSION TITLE
The graph can not self-adaptive.

### DISCUSSION THREAD

**Discussion by saradu1231** [recharts/recharts#3043] [Q&A] [answered]
I try to use your demo on your websites, and it works fine, but when I use it on my data, it can not self-adaptive. The graph has issues.

---------------------------------------------------The graph--------------------------------------------------

- As you can see in the photo, the largest data is around 50,000, but why the highest of YAxis is just 10,000?  
- I see your demo, the barchart below the highest YAxis, and my barchart is over the highest YAxis. Why it is happened?
- Because the graph is not correct, so the 30,000 and the 50,000 looks like same data in the graph. It is confused.

![截图1](https://user-images.githubusercontent.com/83037638/199613022-bb679efc-a4b7-427f-b0f0-53ee5fd8b7b5.png)
![截图2](https://user-images.githubusercontent.com/83037638/199613175-ca82e154-cca0-4dbc-8ff7-1e9a4fe3219a.png)

---------------------------------------------------The code---------------------------------------------------
Code in the attach files.
[graph issue.txt](https://github.com/recharts/recharts/files/9924578/graph.issue.txt)

Thanks for your reply.

**Comment by ckifer** [comment, marked as answer]
Looks like your `domain` is not set correctly

try using 

`<YAxis domain={['dataMin', 'dataMax']} />`
